UserInterface/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Baseclass.Contrib.Nuget.Output" version="2.0.0" targetFramework="net45" />
+  <package id="Baseclass.Contrib.Nuget.Output" version="2.1.0" targetFramework="net45" />
```

Comment:
does this fix the build issue we were seeing in vs2015? I believe we saw errors from an MSBuild.Tasks dll.